README.md

```diff
@@ -1,9 +1,17 @@
 AeroGear Windows Cookbook
 =========================
 
-
 The AeroGear Windows cookbook is a list of recipes to quick start your Windows AeroGear experience.
 
+|                 | Project Info  |
+| --------------- | ------------- |
+| License:        | Apache License, Version 2.0  |
+| Build:          | Visual Studio  |
+| Documentation:  | https://aerogear.org/windows/  |
+| Issue tracker:  | [https://issues.jboss.org/browse/AEROGEAR/](https://issues.jboss.org/browse/AEROGEAR/component/12314947/?selectedTab=com.atlassian.jira.jira-projects-plugin:component-summary-panel)  |

```

Comment:
Let's wait for [AEROGEAR-1650](https://issues.jboss.org/browse/AEROGEAR-1650) in order to get Windows-specific issue tracker.